packages/graphviz.rb

```diff
@@ -0,0 +1,25 @@
+require 'package'
+
+class Graphviz < Package
+  description 'Graphviz is open source graph visualization software.'
+  homepage 'https://www.graphviz.org/'
+  version '2.40.1'
+  source_url 'https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/graphviz.tar.gz'
```

Comment:
Can we get a url that is versioned?  I think this url could be volatile.

Reply:
It's difficult to find that

Reply:
This is the download from the `stable_release_2.40.1` tagged commit hash:
https://gitlab.com/graphviz/graphviz/repository/67cd2e5121379a38e0801cc05cce5033f8a2a609/archive.tar.bz2